src/app/components/Routes/layout.tsx

```diff
@@ -1,17 +1,21 @@
 import React from 'react';
-import { bool, string } from 'prop-types';
 import { NavigationContainer } from '@react-navigation/native';
 import { createStackNavigator } from '@react-navigation/stack';
 
-import { Library, BookDetail, Login } from '@screens';
+import { Library, BookDetail, Login } from '@screens/index';
 import withLoader from '@components/Loader';
+import withCustomHeader from '@components/Header';
 import * as Routes from '@constants/routes';
 
 import { APP_OPTIONS, LOGIN_OPTIONS } from './constants';
 
+interface Props {
+  token: string | null;
```

Comment:
`token?: string;` or you receive null's sometimes?

Reply:
The default value for the token, inside the `globalState`, is null. So I would always get a value: string | null. I thought of setting `undefined` in the `globalState` and set `token?` instead, but as a good practice I don't like to set `undefined` manually.